ansible/roles/prometheus/defaults/main.yml

```diff
@@ -19,8 +19,7 @@ prometheus_configuration:
 
   # Load rules once and periodically evaluate them according to the global 'evaluation_interval'.
   rule_files:
-    # - "first_rules.yml"
-    # - "second_rules.yml"
+    - "rules.yml"
```

Comment:
Will ansible install prometheus in /etc/prometheus by default ?

I'm asking because i'm curious to how the file discovery works

Reply:
Prometheus will use `/etc/prometheus/prometheus.yml` as its standard and default configuration file - Debian doesn't overwrite this. So the rule file will be loaded from `/etc/prometheus/rules.yml`